interfaces/offscreen_render_target.hpp

```diff
@@ -120,6 +120,17 @@ namespace bnb::oep::interfaces
          * @example get_current_buffer_texture()
          */
         virtual rendered_texture_t get_current_buffer_texture() = 0;
+
+        /**
```

Comment:
The same as above. Please review the interfaces before doing something. In general it wasn't expected that these files will be ever changed.